Change examples structure (#436)
asyncapi/asyncapi: versions/2.0.0/asyncapi.md

<a name="messageObjectTags"></a>tags | [Tags Object](#tagsObject) | A list of tags for API documentation control. Tags can be used for logical grouping of messages.
<a name="messageObjectExternalDocs"></a>externalDocs | [External Documentation Object](#externalDocumentationObject) | Additional external documentation for this message.
<a name="messageObjectBindings"></a>bindings | [Message Bindings Object](#messageBindingsObject) | A map where the keys describe the name of the protocol and the values describe protocol-specific definitions for the message.

> <a name="messageObjectExamples"></a>examples | [`any`] | An array with examples of valid message objects. Whatever "any" is, it has to match the type defined on payload. 

Reviewer: ```suggestion
<a name="messageObjectExamples"></a>examples | [`any`] | An array with examples of valid message objects. Whatever "any" is, it MUST match the type defined on the payload. 
```

<a name="messageTraitObjectTags"></a>tags | [Tags Object](#tagsObject) | A list of tags for API documentation control. Tags can be used for logical grouping of messages.
<a name="messageTraitObjectExternalDocs"></a>externalDocs | [External Documentation Object](#externalDocumentationObject) | Additional external documentation for this message.
<a name="messageTraitObjectBindings"></a>bindings | [Message Bindings Object](#messageBindingsObject) | A map where the keys describe the name of the protocol and the values describe protocol-specific definitions for the message.

> <a name="messageTraitObjectExamples"></a>examples | [`any`] | An array with examples of valid message objects. Whatever "any" is, it has to match the type defined on payload. 

Reviewer: ```suggestion
<a name="messageTraitObjectExamples"></a>examples | [`any`] | An array with examples of valid message objects. Whatever "any" is, it MUST match the type defined on the payload. 
```
